Dockerfile

```diff
@@ -20,115 +50,100 @@ RUN apt-get -y update \
-# this makes "apt-key list" output prettier too!
-RUN set -x \
-	&& export GNUPGHOME="$(mktemp -d)" \
-	&& gpg --keyserver ha.pool.sks-keyservers.net --recv-keys DD95CC430502E37EF840ACEEA5D32F012649A5A9 \
-	&& gpg --export DD95CC430502E37EF840ACEEA5D32F012649A5A9 > /etc/apt/trusted.gpg.d/neurodebian.gpg \
-	&& rm -rf "$GNUPGHOME" \
-	&& apt-key list | grep neurodebian
-
-RUN { \
-	echo 'deb http://neuro.debian.net/debian trusty main'; \
-	echo 'deb http://neuro.debian.net/debian data main'; \
-	echo '#deb-src http://neuro.debian.net/debian-devel trusty main'; \
-} > /etc/apt/sources.list.d/neurodebian.sources.list
-
 RUN sed -i -e 's,main *$,main contrib non-free,g' /etc/apt/sources.list.d/neurodebian.sources.list; grep -q 'deb .* multiverse$' /etc/apt/sources.list || sed -i -e 's,universe *$,universe multiverse,g' /etc/apt/sources.list
-    
+
 # Install FSL 5.0.9
 RUN apt-get update && \
     apt-get install -y --no-install-recommends curl && \
-    curl -sSL http://neuro.debian.net/lists/trusty.us-ca.full >> /etc/apt/sources.list.d/neurodebian.sources.list && \
-    apt-key adv --recv-keys --keyserver hkp://pgp.mit.edu:80 0xA5D32F012649A5A9 && \
-    apt-get update && \
-    apt-get install -y fsl-core=5.0.9-4~nd14.04+1
+    curl -sSL http://neuro.debian.net/lists/trusty.us-ca.full >> /etc/apt/sources.list.d/neurodebian.sources.list
 
-# Configure environment
-ENV FSLDIR=/usr/share/fsl/5.0
-ENV FSL_DIR="${FSLDIR}" \
-    FSLOUTPUTTYPE=NIFTI_GZ \
-    PATH=/usr/lib/fsl/5.0:$PATH \
-    FSLMULTIFILEQUIT=TRUE \
-    POSSUMDIR=/usr/share/fsl/5.0 \
-    LD_LIBRARY_PATH=/usr/lib/fsl/5.0:$LD_LIBRARY_PATH \
-    FSLTCLSH=/usr/bin/tclsh \
-    FSLWISH=/usr/bin/wish \
-    FSLOUTPUTTYPE=NIFTI_GZ
+RUN apt-key adv --recv-keys --keyserver hkp://pgp.mit.edu:80 0xA5D32F012649A5A9
 
-RUN echo "cHJpbnRmICJrcnp5c3p0b2YuZ29yZ29sZXdza2lAZ21haWwuY29tXG41MTcyXG4gKkN2dW12RVYzelRmZ1xuRlM1Si8yYzFhZ2c0RVxuIiA+IC9vcHQvZnJlZXN1cmZlci9saWNlbnNlLnR4dAo=" | base64 -d | sh
+RUN apt-get update && \
+    apt-get install -y fsl-core=5.0.9-4~nd14.04+1
+
+RUN apt-get build-dep -y gridengine && apt-get update -y
 
 # Install Connectome Workbench
 RUN apt-get update && apt-get -y install connectome-workbench=1.2.3-1~nd14.04+1
 
-ENV CARET7DIR=/usr/bin
 
-# Install HCP Pipelines and MSM binaries
-WORKDIR /opt
-RUN apt-get -y update \
+# Install HCP Pipelines
+RUN apt-get update \
     && apt-get install -y --no-install-recommends python-numpy && \
-    wget -q https://github.com/Washington-University/Pipelines/archive/v3.17.0.tar.gz -O pipelines.tar.gz && \
-    tar zxf pipelines.tar.gz && \
-    mv /opt/Pipelines-* /opt/HCP-Pipelines && \
-    rm pipelines.tar.gz && \
-    wget -q https://www.doc.ic.ac.uk/~ecr05/MSM_HOCR_v1/MSM_HOCR_v1-download.tgz -O MSMs.tar.gz && \
-    tar zxf MSMs.tar.gz && \
-    rm MSMs.tar.gz && \
-    mv MSM_HOCR_v1/Ubuntu /opt/HCP-Pipelines/MSMBinaries && \
-    rm -rf MSM_HOCR_v1
```

Comment:
Is there a reason for not installing MSM?

Reply:
No reason, let's go with your version.